index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>feelgood</title>
+    <link rel="stylesheet" href="main.css"/>
+    <script src = "app.js"></script>
```

Comment:
To fix the issue with nothing happening, you need to include jQuery as a script (based off of [this SO answer](https://stackoverflow.com/a/2195167)). 

Changing this kicks off a different issue that is CORS-related; more comments coming.

Edit: CORS problems were solved by enabling CORS in the API and redeploying the API. So, basically, voodoo.

![console_debug](https://user-images.githubusercontent.com/29528527/75943663-77e04a00-5e4a-11ea-933a-fab0f13f14b6.png)

```suggestion
    <script src = "app.js"></script>
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```